docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md

```diff
@@ -159,6 +159,8 @@ The options in `Global` section are used for openstack-cloud-controller-manager
 
 ###  Networking
 
+* `trunk-support-enabled`
+  Indicates whether or not IP addresses attached via trunk is supported. Default: false
```

Comment:
As noted, this should not be required. The list of returned ports already includes subports. If they are present then trunk support is present and in use. If they are not present then trunk support is not in use.

Configuration is an API. We shouldn't add it where it is not required.

Reply:
Sure.